src/views/Tournament/Tournament.tsx

```diff
@@ -129,6 +129,8 @@ interface TournamentInterface {
     opengotha_standings?: boolean;
 }
 
+type EditSaveState = "none" | "saving" | "reload";
```

Comment:
Not sure this is really providing any type safety benefit when using React's useState. Is there some way to declare a type in `useState` that I missed?

Reply:
Yep like this: `React.useState<EditSaveState>("none")`

Reply:
(Once that's in you shouldn't nee dto do any of the "as EditSaveState" stuf fin either the useState call or the setEditSaveState)

Reply:
Yeah, and there are examples of that nearby too... done in 3cb14c68159b83fb4c7c4bf20b5483011e5e54f8.